src/components/Buttons.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+interface Props {
+  loadAll: () => void;
+  loadFirstFive: () => void;
+  loadRedGoods: () => void;
+}
+const Buttons: React.FC<Props> = ({ loadAll, loadFirstFive, loadRedGoods }) => {
+  return (
+    <div className="container">
+      <button
+        className="button"
+        type="button"
+        onClick={() => loadAll()}
+      >
+        Load All goods
+      </button>
+      <button
+        type="button"
+        className="button"
+        onClick={() => loadFirstFive()}
+      >
+        Load 5 first goods
+      </button>
+      <button
+        className="button"
+        type="button"
+        onClick={() => loadRedGoods()}
+      >
+        Load red goods
+      </button>
```

Comment:
would be great if you create an array that contains all info about each button and then you map that array and write button only once (since all buttons have the same structure)